README.md

```diff
@@ -6,20 +6,19 @@ This repo is the starting point for building .NET 6 from source. Instructions fo
 
 ## Prerequisites
 
-The dependencies for building .NET 6.0 from source can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md).
+The dependencies for building .NET from source can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/requirements/linux-requirements.md). It may also be helpful to reference the [dotnet-buildtools-prereqs-docker](https://github.com/dotnet/dotnet-buildtools-prereqs-docker) repo, which we use to build our container images for development and CI.
 
-## Building .NET 6.0
+## Building .NET 7.0 and .NET 6.0
```

Comment:
Can we simply mention it as .NET to be future proof?

Next section (3.1) will be removed soon after it is out of support.

Reply:
I intentionally mentioned 6.0 and 7.0 here since the instructions will end up changing for 8.0. If you or @MichaelSimons have other suggestions, I'd be happy to hear them.